packages/morph/lib/dom-helper.js

```diff
@@ -153,7 +153,11 @@ prototype.setAttribute = function(element, name, value) {
 };
 
 prototype.setAttributeNS = function(element, namespace, name, value) {
-  element.setAttributeNS(namespace, name, String(value));
+  if (element.setAttributeNS) {
```

Comment:
Is this feature detection need on an element-by-element basis, or can we treat it as a browser feature detect?

```js
// I'm sure this detect can be written better, see the top of the file for others
if (document && document.createElement('div').setAttributeNS)  {
  prototype.setAttributeNS = function(element, namespace, name, value) {
    element.setAttributeNS(namespace, name, String(value));
  }
} else {
  prototype.setAttributeNS = function(element, namespace, name, value) {
    element.setAttribute(name, String(value));
  }
}
```